modules/wmco-prerequisites.adoc

```diff
@@ -7,6 +7,12 @@
 
 The following information details the supported platform versions, Windows Server versions, and networking configurations for the Windows Machine Config Operator. See the vSphere documentation for any information that is relevant to only that platform.
 
+[IMPORTANT]
+====
+Because link:https://techcommunity.microsoft.com/t5/containers/important-update-deprecation-of-docker-virtual-machine-images/ba-p/3646272[Microsoft has stopped publishing Windows Server 2019 images with
+Docker], Red Hat no longer supports Windows Azure for WMCO releases older than version 6.0.0. For WMCO 5.y.z and earlier, Windows Server 2019 images must have Docker pre-installed. WMCO 6.0.0 and later uses containerd as the runtime. You can can either upgrade to {product-title} 4.11, which uses WMCO 6.0.0.
```

Comment:
s/You can can either upgrade/You can upgrade/g
(I'm not native speaker, but isn't either used when you are about to give two options? ex: "You either chose going to the beah, or going to the sea". However, I only see one single alternative)